ops/tf-modules/warehouse-cluster/main.tf

```
@@ -10,7 +10,7 @@ locals {
       machine_type       = "e2-standard-2"
```

Let's scale this up actually. 
```suggestion
      machine_type       = "e2-standard-4"
```